playbooks/scenario4_image_setup.yml

```diff
@@ -18,14 +18,12 @@
           totalcpu: "true"
           fielddrop:
             - "time_*"
-  tasks:
-  - name: install nginx
-    become: yes
-    apt:
-      name: ['nginx', 'nginx-extras']
 
-  - name: start nginx
-    become: yes
-    service:
-      name: nginx
-      state: started
+- name: Start test Server
```

Comment:
Is it needed to start server during image creation?

Reply:
i think it's a good idea, we can catch a problem, if it occur, with server while image creation. It's better than face this when infrastructure will be already deployed

Reply:
Seems legit